cran-comments.md

```diff
@@ -1,14 +1,22 @@
 ## Resubmission
 
 This is a resubmission. In this version I have:
-* Autolinked DOI in `DESCRIPTION`
-* Made fixes to R chunks and minor edits in `README.md` 
-* Changed dataset_name to database_name
-* Added a simple `rec_with_table()` function call that uses `data` variable name 
-as `database_name` as well as default `variables` and `variable_details` that
-come  with the package.
-* Modified tests and examples to use the new simplified rec_with_table call
-* Modified vignettes to reflect use of new `rec_with_table()` call
+* Enabled the use of message() to suppress messages on console
+* Added quotation marks to 'sjmisc' in `DESCRIPTION`
+* Added food_insecurity_der as a new derived variable using
+`food_insecurity_fun1()`, `food_insecurity_fun2()`, and `food_insecurity_fun3()`
```

Comment:
When u redo food_insecurity with variable details make sure to refactor this. Also consider adding Note to food_insecurity

Reply:
See my commit edits